### ISSUE TITLE
Style scrollbars for the dark theme

### ISSUE THREAD

**Issue by jouni** [vaadin/vaadin-lumo-styles#39]
When using `theme="dark"`, we should style native browser scrollbars when possible, to make them blend in nicely.

**Comment by berndhopp** [comment]
any progress here?

**Comment by jouni** [comment]
No, haven’t had any time to consider this. If you are interested, you could do research about the current possibilities in various browsers and platforms at the moment, what can and can’t be styled.

**Comment by lkraav** [comment]
My recent research showed there is no universal CSS mechanism available at this point.

https://www.filamentgroup.com/lab/scrollbars/

I did find `scrollbar-width` usable, but also need to add a lot of `-webkit`-prefixed stuff, see https://github.com/conversionxl/aybolit/blob/cxl-lumo-styles-0.2.2/packages/cxl-ui/scss/cxl-institute-layout.scss#L33-L39

**Comment by berndhopp** [comment]
thank you
